common/src/main/java/org/dash/wallet/common/ui/address_input/AddressInputFragment.kt

```diff
@@ -210,7 +210,6 @@ abstract class AddressInputFragment : Fragment(R.layout.fragment_address_input)
                     )
                 }
             )
-            notifyItemRangeInserted(0, source.size)
```

Comment:
this was causing a crash when the list was refeshed.  With the adapter that is used, we don't need to notify about updates.